src/runtime/google-analytics-event-listener.ts

```diff
@@ -111,6 +110,7 @@ export class GoogleAnalyticsEventListener {
         'event_category': gaEvent.eventCategory,
         'event_label': gaEvent.eventLabel,
         'non_interaction': gaEvent.nonInteraction,
+        'configurationId': event.configurationId || '',
```

Comment:
I think it's redundant information fo GA logging. The key needs to be manually configured as a custom dimension to be processed.

Reply:
Makes sense. Removed.